src/digital/v2_compat.rs

```diff
@@ -153,6 +159,7 @@ mod tests {
         }
     }
 
+    #[cfg(feature = "unproven")]
```

Comment:
... and this

Reply:
same as https://github.com/rust-embedded/embedded-hal/pull/164#discussion_r347004908